scripts/annotation.js

```diff
@@ -37,18 +37,37 @@ $(document).ready( ()=> {
                 choice = true;
                 break;
         }
+        add_annotation()
+    };
+
+    image_div.click( ()=> {
+        // nextImage();
+    });
+    
+    //takes image_div and applies "swipeleft" event 
+    //to the image
+    image_div.hammer().on("swipeleft", function() {
+        choice = false;
+        add_annotation()
+    });
+    
+    //Takes image_div and applies "swiperight" event
+    //to the image
+    image_div.hammer().on("swiperight", function(){
+        choice = true;
+        add_annotation()
+    });
+
+    function add_annotation() {
```

Comment:
We may have to re-assess scope here so we actually pass all the variables into `add_annotation` instead of relying on an outer scope.